Modules/Log/Get-RegistryKeyValData.ps1

```diff
@@ -0,0 +1,169 @@
+<#
+.SYNOPSIS
+Get-RegistryKeyValData.ps1 retrieves the value of the provided key as
+well as the last modified time of the key.
+
+.NOTES
+Next line needed by Kansa.ps1 for proper handling of this script's data
+OUTPUT tsv
+
+HKEY_LOCAL_MACHINE
+HKEY_USERS
+HKEY_CURRENT_USER is a subkey of HKEY_USERS
+HKEY_CURRENT_CONFIG is a subkey (HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Hardware Profiles\Current)
+HKEY_CLASSES_ROOT is a subkey (HKEY_LOCAL_MACHINE\SOFTWARE\Classes)
+#>
+param(
+[string]$Key
+)
+
+$Error.Clear()
+$ErrorActionPreference = "SilentlyContinue"
+
+## The following code is from
+## Name: Get-RegistryKeyTimestamp
```

Comment:
Setting this will keep any of your catch blocks from doing anything.

```PowerShell
$Orrig_EAP = $ErrorActionPreference
$ErrorActionPreference = "SilentlyContinue"

Write-Host "before try"

try
{
    Write-Host "In try"
    Write-Error "Bad stuff happened"
    Write-Host "After error"
}
catch
{
    Write-Host "In catch"
}
finally
{
    Write-Host "In finally"
    $ErrorActionPreference = $Orrig_EAP
}

Write-Host "after try"
```

```
before try
In try
After error
In finally
after try
```